twenty/umbrel-app.yml

```
@@ -0,0 +1,37 @@
+manifestVersion: 1
+id: twenty
+name: Twenty
+tagline: An adaptable CRM solution for seamless team collaboration and growth
+category: files
+version: "0.50"
```

Use of the full version.
```suggestion
version: "0.50.0"
```